R/chromote-fbref.R

```diff
@@ -0,0 +1,55 @@
+
+#' @source <https://github.com/tidyverse/rvest/blob/chromote/R/chromote.R>
```

Comment:
i basically just borrowed some code from the `chromote` branch of `rvest`, which enables selenium like behavior with minimal dependencies (only new deps are `chromote` and `R6`). without chromote we could only grab the team table at the top of a page like [this](https://fbref.com/en/comps/9/shooting/Premier-League-Stats). `chromote` enables us to get the player table

Reply:
i expect that `rvest`'s `chromote` branch will eventually be merged, at which point we can remove this code and rely on `rvest`'s functionality

Reply:
Really cool!